iconservice/prep/engine.py

```diff
@@ -597,8 +638,17 @@ def handle_set_prep(cls, context: 'IconScoreContext', params: dict):
 
         # Update registration info
         dirty_prep.set(**kwargs)
+
         context.put_dirty_prep(dirty_prep)
 
+    @classmethod
+    def _validate_node_address(cls,
+                               context: 'IconScoreContext',
+                               key: 'Address'):
+
+        if key in context.prep_address_converter.node_address_mapper:
```

Comment:
I think that calling a method seems better than accessing to the property "node_address_mapper" directly.